ui/app/src/components/GraphiQL.tsx

```diff
@@ -72,34 +72,22 @@ function Graphi(props: GraphiQLProps) {
       const variables = encodeURIComponent(JSON.stringify(graphQLParams['variables']));
 
       url += `?query=${query}&variables=${variables}`;
-
-      return fetch(url, {
-        method: 'get',
-        headers: { 'Content-Type': 'application/json' }
-      }).then(function (responseBody: any) {
-        try {
-          return responseBody.json();
-        } catch (error) {
-          return responseBody;
-        }
-      });
-
-    }else{
-      return fetch(url, {
-        method: 'post',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(graphQLParams),
-      }).then(function (responseBody: any) {
-        try {
-          return responseBody.json();
-        } catch (error) {
-          return responseBody;
-        }
-      });
     }
+
+    return fetch(url, {
+      method,
+      headers: { 'Content-Type': 'application/json' },
+      body: method === 'post' ? JSON.stringify(graphQLParams) : ''
```

Comment:
This way, the body attribute is still present in the request. Do it as previously requested.

Reply:
done!